setup.cfg

```diff
@@ -26,4 +26,42 @@ url = http://ccdproc.readthedocs.io/
 edit_on_github = False
 github_project = astropy/ccdproc
 
+# PEP8 errors/warnings:
+# (partially) taken from
+# https://pycodestyle.readthedocs.io/en/latest/intro.html#error-codes
+
+# E101 - mix of tabs and spaces
+# E111 - 4 spaces per indentation level
+# E112 - 4 spaces per indentation level
+# E113 - 4 spaces per indentation level
+# E221 - multiple spaces before operator
+# E222 - multiple spaces after operator
+# E223 - tab before operator
+# E224 - tab after operator
+# E225 - missing whitespace around operator
+# E241 - multiple whitespace after ','
+# E242 - tab after ‘,’
+# E251 - unexpected spaces around keyword / parameter equals
+# E271 - multiple spaces after keyword
+# E272 - multiple spaces before keyword
+# E303 - too many blank lines
+# E304 - blank lines found after function decorator
+# E502 - the backslash is redundant between brackets
+# E703 - statement ends with a semicolon
+# E901 - SyntaxError or IndentationError
+# E902 - IOError
+# W191 - indentation contains tabs
+# W291 - trailing whitespace
+# W292 - no newline at end of file
+# W293 - blank line contains whitespace
+# W391 - blank line at end of file
+
+[pycodestyle]
+select = E101,E111,E112,E113,E221,E222,E223,E224,E225,E241,E242,E251,E271,E272,E303,E304,E502,E703,E901,E902,W191,W291,W292,W293,W391
+
+# E402 module level import not at top of file
+# We need to ignore this as the .conf has to be set in the __init__.py
+# before the modules are imported.
+ignore = E402
```

Comment:
How often do you have this in ccdproc? We have many configs in astroquery, this it's always ignored, but you may not want to copy that.